dist/sgx/setup-new-powhsm

```diff
@@ -3,21 +3,37 @@
 ROOT_DIR=$(realpath $(dirname $0))
 source $ROOT_DIR/scripts/print_utils
 
+SERVICE_NAME=powhsmsgx
+
 # Require superuser, since we need to install a service in the host
 if ! [ "$(id -u)" == "0" ]; then
     print_info "Please run with sudo."
     exit 1
 fi
 
+print_info "Welcome to the SGX powHSM Setup for RSK"
+echo
+
+$ROOT_DIR/scripts/install_service
+if [ $? -ne 0 ]; then
+	print_error "Error during the powhsm service installation, aborting"
+	exit 1
+fi
+
```

Comment:
Why is there a second argument passed in to the `run_with_docker` script?

Reply:
It's the optional `-v` flag. This logic didn't change, if we pass a `-v`, `docker build` runs without the `-q` flag:

```
if [[ "$2" != "-v" ]]; then
    QUIET="-q"
    echo -e "Run with '-v' if you want to see progress detail\e[94m"
fi
docker build -t $DOCKER_IMAGE $DIST_DIR $QUIET
echo -e "\e[96mDocker image build done.\e[0m"
echo
```